Directory.Build.rsp

```diff
@@ -0,0 +1,15 @@
+#------------------------------------------------------------------------------
+# This file contains command-line options that MSBuild will process as part of
+# every build, unless the "/noautoresponse" switch is specified.
+#
+# MSBuild processes the options in this file first, before processing the
+# options on the command line. As a result, options on the command line can
+# override the options in this file. However, depending on the options being
+# set, the overriding can also result in conflicts.
+#
+# NOTE: The "/noautoresponse" switch cannot be specified in this file, nor in
+# any response file that is referenced by this file.
+#------------------------------------------------------------------------------
+/m
+/verbosity:minimal
```

Comment:
I'd prefer that we put all these flags in the `build.cs`.

Reply:
I'm not familiar with BuildNuke, which seems to be what build.cs is using. You're welcome to take these ideas from the PR and apply it yourself.
FWIW though, I never would have benefitted from the changes had you set these flags there, because as an OSS contributor I just used `dotnet build` and/or `msbuild` in your repo to build it. I tend to use the standard tools as much as possible and keep my repos working with those for others. The .rsp file applies to all msbuild CLI invocations. I don't know if it would impact your build.cs-spawned builds at all.

Reply:
All of these settings are defaults in VS for Windows too. This .rsp file addition is just to improve the CLI experience for those of us that use that (in addition to IDEs).

Reply:
I think this parameters are already the default (at least on my machine):

If I am running `dotnet build -clp` (or any other unknown/unfulfilled cli switch) I get the following output:
```cmd
MSBUILD : error MSB1037: Specify one or more parameters for the console logger if using the -consoleLoggerParameters switch
    Full command line: 'C:\Program Files\dotnet\sdk\7.0.102\MSBuild.dll -maxcpucount -verbosity:m -restore -consoleloggerparameters:Summary -clp'
```
-maxcpucount (which is the long term for `-m`)
-verbosity:minimal and
-consoleloggerparameters:Summary (which is the long term for -clp:Summary)

The only think which is missing is the `ForceNoAlign`

Reply:
Can confirm: also on my mac this seems to be the default

Reply:
Yes, for `dotnet build` they are the default (which shows they're good!) but they are _not_ the default for `msbuild.exe`.

Reply:
Is anybody still using `msbuild.exe`?